datetime/time.rs

```diff
@@ -22,7 +22,9 @@ use plib::PROJECT_NAME;
 #[command(
     version,
     about = gettext("time - time a simple command or give resource usage"),
-    help_template = gettext("{about-with-newline}\nUsage: {usage}\n\nArguments:\n{positionals}\n\nOptions:\n{options}")
+    help_template = gettext("{about}\n\nUsage: {usage}\n\nArguments:\n{positionals}\n\nOptions:\n{options}"),
+    disable_help_flag = true,
+    disable_version_flag = true,
```

Comment:
Why disable?   Is this necessary for gettext-rs to translate the "Print help" string?

Reply:
Yes